required/tools/multicol.dtx

```diff
@@ -92,13 +92,13 @@
 %<package>\DeclareRelease{v1.8}{2019-10-01}{multicol-2019-10-01.sty}
 %<package>\DeclareCurrentRelease{}{2021-11-15}
 %<package>
+%<package>\NeedsTeXFormat{LaTeX2e}[2018-04-01]
+%<package>
```

Comment:
Is `\NeedsTeXFormat{LaTeX2e}[2018-04-01]` really needed here? It was changed from `\NeedsTeXFormat{LaTeX2e}[1997/12/01]` in https://github.com/latex3/latex2e/commit/921f5fb086325f3e1ff2a36efd7c10e4c7ccdee5 but only because the package rollback mechanism was introduced in that commit. Maybe this should be changed back into `\NeedsTeXFormat{LaTeX2e}[1997-12-01]`?

Reply:
good point. It probably should now

Reply:
On second thought I don't think it should (or needs to be). The file  does use the rollback commands which aren't there if you really try use it with a release prior to 2018. Of course it now comes after  getting undefined csname then, but as a record the date is still not wrong.

In the rollback situation it doesn't apply because the other file is loaded instead.

Reply:
Wouldn't `\NeedsTeXFormat{LaTeX2e}[2018-04-01]` then also have to be added to all other files that contain rollback declarations? These do not yet contain such a line, although they do not really differ from `multicol` in terms of rollback.

On the other hand, I think that `\NeedsTeXFormat{LaTeX2e}[2018-04-01]` only has a very limited benefit. It only comes into effect if you use the new `multicol` is together with a pre-2018 kernel _and_ continue after two `Undefined control sequence` errors (`\DeclareRelease` and `\DeclareCurrentRelease`) ...